blueocean-dashboard/src/main/js/components/RunDetailsPipeline.jsx

```diff
@@ -20,8 +22,15 @@ const { string, object, any, func } = PropTypes;
 
 export class RunDetailsPipeline extends Component {
     componentWillMount() {
-        if (this.props.fetchNodes) {
-            this.props.fetchNodes(this.generateConfig(this.props));
+        const { fetchNodes, fetchLog, result } = this.props;
+        const mergedConfig = this.generateConfig(this.props);
+
+        if (result && result._class === 'io.jenkins.blueocean.service.embedded.rest.PipelineRunImpl') {
```

Comment:
yeah once day we need to extract this into an extension point or capabilities, but I think this is fine for now.